bin/inference/pycbc_inference_start_from_dynesty

```diff
@@ -0,0 +1,46 @@
+#!/bin/env python
+""" Convert dynesty output file to emcee_pt compatible start format
+"""
+
+import argparse
+import numpy
+from numpy.random import choice
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input-file')
+parser.add_argument('--output-file')
+parser.add_argument('--ntemps', type=int)
+parser.add_argument('--nwalkers', type=int)
+args = parser.parse_args()
+
+# populate an emcee start file with
+# values chosen from a dynesty file
+# each temperature and walker will get a random
+# point from the dynesty output
+
+ntemps = args.ntemps
+nwalkers = args.nwalkers
+
+import h5py
+f = h5py.File(args.input_file, 'r')
+
+nsample = len(f['samples'][f['samples'].keys()[0]])
+
+# These are the ids we'll use for the temps / walkers
+use = choice(nsample, replace=False, size=ntemps * nwalkers)
+
+o = h5py.File(args.output_file, 'w')
+for k in f['samples']:
+    data = f['samples'][k][:][use]
+    o['samples/' + k] = data.reshape(ntemps, nwalkers, 1)
+
+o.attrs['filetype'] = 'emcee_pt_file'
+o.attrs['static_params'] = []
+o.attrs['variable_params'] = f['samples'].keys()
```

Comment:
Set the output's `variable_params` to `f.variable_params`; otherwise you'll end up with things like the `loglikelihood` in there. That may or may not be an issue later on, but better to err on safety.